plugins/text-freetype2/CMakeLists.txt

```diff
@@ -1,89 +1,49 @@
 project(text-freetype2)
 
-if(DISABLE_FREETYPE)
-	message(STATUS "Freetype text plugin disabled")
-	return()
-endif()
+option(ENABLE_FREETYPE "Enable Freetype text plugin" ON)
```

Comment:
Nitpick:  FreeType.  Here and other places.

Reply:
The finder is called "FindFreetype" and it's ID is `Freetype::Freetype` following convention. I changed the spelling of the log messages, but for all intents and purposed (and internal CMake convention) it has to be "Freetype".

Reply:
Got it.  Thanks for explaining.